cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc

```diff
@@ -45,9 +45,11 @@ NOTE: The details and functionality listed below are a preview and the actual re
 //RLP-112353
 | By the end of 2023 calendar year, BridgecrewCLI including GitHub Action, CircleCI Orb, and container, will be deprecated.  Plan to transition smoothly to Checkov and its compatible plugins to minimize disruption.
 
-|The following endpoints are deprecated in release 23.8.1 and will be sunset in release 23.9.1:
+|The following endpoints are deprecated in release 23.8.1 and will be sunset in release 23.9.2:
```

Comment:
Is this right heading?

Reply:
@manukumar6  - please help confirm

Reply:
Yes, Matangi. The complete note is there: https://github.com/PaloAltoNetworks/prisma-cloud-docs/blame/8c981c3c13804575fc62565e3a8f9d2ae7554ca9/cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc